framework/packages/abstract-core/src/objects/chain_name.rs

```diff
@@ -32,6 +37,20 @@ impl ChainName {
         Ok(chain_name)
     }
 
+    /// Gets the chain name from a raw ans-entry.
+    pub fn from_ibc_entry(entry: String) -> AbstractResult<Self> {
```

Comment:
This doesn't seem to be used anywhere? What's the goal of this function?

Reply:
It's used in the modules folder. It's used to get "juno" from "juno>wyndex", to know what app to route to

Reply:
Let's remove it. I don't like that it accepts a String type. We should have some overarching type for this (`Trace`?)

Because assets also have it... and account ids as well...

`TraceString`?